src/metabase/notification/payload/impl/card.clj

```diff
@@ -8,27 +8,28 @@
    [toucan2.core :as t2]))
 
 (mu/defmethod notification.payload/payload :notification/card
-  [{:keys [creator_id alert] :as _notification-info} :- notification.payload/Notification]
-  (let [card_id (:card_id alert)]
-    {:card_part   (notification.execute/execute-card creator_id card_id
-                                                     ;; for query_execution's context purposes
-                                                     ;; TODO: check whether we can remove this or name it?
-                                                     :pulse-id (:id alert))
-     :card        (t2/select-one :model/Card card_id)
-     :style       {:color_text_dark   channel.render/color-text-dark
-                   :color_text_light  channel.render/color-text-light
-                   :color_text_medium channel.render/color-text-medium}
-     :alert       alert}))
+  [{:keys [creator_id payload subscriptions] :as _notification-info} :- notification.payload/Notification]
+  (let [card_id (:card_id payload)]
+    {:card_part       (notification.execute/execute-card creator_id card_id
+                                                         ;; for query_execution's context purposes
+                                                         ;; TODO: check whether we can remove this or rename it?
+                                                         :pulse-id (:id payload))
+     :card             (t2/select-one :model/Card card_id)
+     :style            {:color_text_dark   channel.render/color-text-dark
+                        :color_text_light  channel.render/color-text-light
+                        :color_text_medium channel.render/color-text-medium}
+     :notification_card payload
```

Comment:
I have some convection here, on one hand I'd love to name this `:payload` but since this whole map is already nested in side a :payload key(see notification.payload/notification-payload method), so having :payload nested inside :payload is a bit ugly